coralproject/talk
# Hide Pre-Mod tab sometimes

## What does this PR do?

if Premoderation is enabled globally or there are pre-mod comments, then show the Pre-Mod tab. Otherwise, hide it.

## How do I test this PR?

- enable pre-moderation globally
- make a comment and do not moderate it
- Pre-mod tab should be there
- disable pre-moderation globally
- Pre-mod tab should still be there
- moderate the outstanding comment
- POOF! the Pre-Mod tab is gone.
